syntax = "proto3";

package remote_agent;

// Helper message for the paired filepath and mime_type
message FileAttachment {
  string local_filepath = 1;
  string mime_type = 2;
}

message MailTo {
  // Optional account name
  optional string account_name = 1;
  
  // Required subject
  string subject = 2;
  
  // Required body
  string body = 3;
  
  // List of file attachments with local filepath and mime type
  repeated FileAttachment file_list = 4;
}

